src/Centreon/Application/Validation/CentreonValidatorFactory.php

```diff
@@ -73,11 +74,16 @@ public function getInstance(Constraint $constraint)
 
         if (!isset($this->validators[$className])) {
             if (class_exists($className)) {
-                // validator as a class with dependencies
-                $this->validators[$className] = new $className(new ServiceLocator(
-                    $this->container,
-                    method_exists($className, 'dependencies') ? $className::dependencies() : []
-                ));
+                // validator as a class with dependencies from centreon
+                if (in_array(CentreonValidatorInterface::class, class_implements($className))) {
```

Comment:
please use the same approach as here: https://github.com/centreon/centreon/blob/master/src/CentreonRemote/Infrastructure/Service/ExporterService.php#L18

Reply:
thanks